addons/website/static/src/snippets/s_alert/options.js

```diff
@@ -0,0 +1,27 @@
+odoo.define('website.s_alert_options', function (require) {
+'use strict';
+
+var options = require('web_editor.snippets.options');
+
+options.registry.Alert = options.Class.extend({
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    selectType: function (previewMode, widgetValue, params) {
+        // compatibility: we need `alert-delta` to be removed, which will be the
+        // case by selecting another color with the colorpicker.
+        let widget = null;
+        // TODO (review) This will not work for every color, need QSM fix/imp?
+        this.trigger_up('user_value_widget_request', {
+            name: 's_alert_colorpicker',
+            onSuccess: _widget => widget = _widget,
+        });
+        const eventName = previewMode === 'reset' ? 'mouseleave' : previewMode ? 'mouseenter' : 'click';
+        widget.$el.find(`[data-color="${widgetValue}"]`).trigger(eventName);
```

Comment:
Well the better way gotta be find now, this is not working anymore since I rebased Tuesday :zzz: 
The select is not closing anymore when clicked because of this line 
```javascript
widget.$el.find(`[data-color="${widgetValue}"]`).trigger(eventName);
```
Any hints? Probably something deployed recently

Reply:
Don't know, I'll test

Reply:
Not needed anymore with the `data-trigger` imp